css/styles.css

```diff
@@ -0,0 +1,100 @@
+@import url('https://fonts.googleapis.com/css2?family=Playfair+Display&display=swap');
+
+@font-face {
+  font-family: PoppinsRegular;
+  src: url(../resources/poppins.regular.ttf);
+}
+
+@font-face {
+  font-family: PoppinsLight;
+  src: url(../resources/poppins.light.ttf);
+}
+
+@font-face {
+  font-family: PoppinsSemiBold;
+  src: url(../resources/poppins.semibold.ttf);
+}
+
+*,
+*::after,
+*::before {
+  margin: 0;
+  padding: 0;
+  box-sizing: border-box;
+}
+
+.playfairFont {
+  font-family: Playfair Display, sans-serif;
+  font-size: 50px;
+  font-weight: 400;
+  line-height: 50px;
+  letter-spacing: 0;
+  text-align: left;
+}
+
+.paragraphStyle {
+  font-family: PoppinsRegular, sans-serif;
+  font-size: 14px;
+  font-weight: 400;
+  line-height: 21px;
+  letter-spacing: -0.01em;
+  text-align: left;
+  padding-top: 15px;
+  padding-bottom: 15px;
+}
+
+.buttonDeco {
+  background-color: #1a2236;
+  color: white;
+  font-family: PoppinsSemiBold, sans-serif;
+  font-size: 19px;
+  font-weight: 500;
+  line-height: 24px;
+  letter-spacing: 0;
+  text-align: center;
+  padding: 15px 70px;
```

Comment:
- [ Required ] I think it would be better if the `padding` in your design is consistent with the padding on the Figma template. It appears that the left and right paddings on this button are too wide compared to the Figma template, on the Figma design the padding on the button is as shown below please rectify this.

```CSS
  padding: 12px 42px;
```
## Figma template
<img src="https://user-images.githubusercontent.com/16392046/227230703-438edc44-d5db-448c-bb16-2332e720f10d.png" width="375px" />

## Your design
![Screenshot from 2023-03-23 17-13-20](https://user-images.githubusercontent.com/16392046/227231108-c6031864-1efd-4843-8f98-8c5e12dbfb35.png)